injection/sharedmain/main.go

```diff
@@ -148,21 +150,19 @@ func EnableInjectionOrDie(ctx context.Context, cfg *rest.Config) context.Context
 
 	ctx, informers := injection.Default.SetupInformers(ctx, cfg)
 
-	// Start the injection clients and informers.
-	logging.FromContext(ctx).Info("Starting informers...")
-	go func(ctx context.Context) {
+	go func() {
```

Comment:
Do we really need this go function? I think it is safer to let it finishes.

Reply:
This will be deleted asap after knative rolls to the new one, shared main now calls down to the new version. this method is not changed and is legacy.

Reply:
My original question is about using go func or not in the old function, but I guess the whole function will be removed anyway.